src/modules/RootPart.rb

```diff
@@ -1240,6 +1240,25 @@ def MountFSTab(fstab, message)
         )
       end
 
+      efivars_path = "/sys/firmware/efi/efivars"
+      if ::File.exist?(efivars_path)
+        if MountPartition(efivars_path, "efivarfs", "efivarfs") == nil
+          AddMountedPartition(
+            { :type => "mount", :device => "efivarfs", :mntpt => efivars_path }
+          )
+        end
+      end
+
+      if SCR.Execute(
+          path(".target.mount"),
+          ["/run", Installation.destdir + "/run", Installation.mountlog],
+          "--bind"
+         )
+        AddMountedPartition(
+          { :type => "mount", :device => "none", :mntpt => "/run" }
+        )
+      end
```

Comment:
This method already have 178 lines. I am not much happy to see it growing even more.
Is possible to move this special mounts to own method?
And when separate method exists, is it possible to write even simple smoke test for it? I will feel more confident in code if it is tested.

Reply:
thanks